pkg/file/filereader_test.go

```diff
@@ -64,18 +64,3 @@ func TestFileReaderParseGood(t *testing.T) {
 
 	assert.Nil(t, actual)
 }
-
```

Comment:
This was causing a circular dependency and these test are tested elsewhere. Long range, I think we should try to get rid of these files as much as we can